recipe/meta.yaml

```diff
@@ -1,39 +1,61 @@
 {% set version = "0.26.0" %}
-{% set pypi_package_name = "u8darts" %}
 
 package:
-  name: u8darts
+  name: darts-packages
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ pypi_package_name[0] }}/{{ pypi_package_name }}/{{ pypi_package_name }}-{{ version }}.tar.gz
-  sha256: 724878deb4cf6ccac7bd20513065fbcace9857839a29e3902ea1aa14cc10484d
+  # it seems that in the pypi package the files are older than the one tagged as 0.26.0
+  # and anyway some tests fail. Also this tagged version is old by two commits, but it works.
+  url: https://github.com/unit8co/darts/archive/refs/tags/0.26.0.tar.gz
+  sha256: c4d895fbf44baba559f9639000f9da0cb70c28bea98f650805252b91f502ea01
+  patches:
+    # For py38 and py39 one test fails after the change in this PR:
+    # https://github.com/unit8co/darts/pull/1932/files
+    # in the file darts/tests/test_timeseries.py
+    # 
+    # For some reason the selectors [py38 or py39] and [py<310] do
+    # not work, so the patch is applied for all Python versions.
+    #
+    # In this patch we restore one line reverting to the
```

Comment:
These comments should be put into the patches IMO.

Reply:
done